_unittest/test_08_Primitives3D.py

```diff
@@ -1308,6 +1308,9 @@ def test_67_assign_material(self):
         assert self.aedtapp.modeler[cyl1].material_name == "aluminum"
         assert self.aedtapp.modeler[cyl2].material_name == "aluminum"
 
+        assert self.aedtapp.modeler[box1].material_name == "pec"
```

Comment:
@IreneWoyna remove these assert statements from here. They will fail in Primitives3D